### PR TITLE
Update of the Readme

### PR DESCRIPTION
**Pull request by ojokenobi**
I updated parts of the readme file, i was thinking about adding the discord bot's icon, but im not too sure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
+ Has youtube music, custom (user defined) dj roles. 
 
-# Prerequisites
+## Prerequisites
 > jdk8/jdk11/jdk16 
 > (jdk11 recommended)
 
 > A minecraft server running bukkit/spigot/paper
-> (skigot recommended for long term usage, paper for short term)
+> (spigot recommended for long term usage, paper for short term)
```
Comment (9382) [inline README.md:10]: This is intentional, not a spelling mistake. Change this back to skigot

File: README.md [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 
-# Installation
-> Move everything in the plugins folder into your minecraft server plugin folder
+## Installation
+If you need help installing any of the servers listed above, refer to the guides for 
+[Spigot](https://www.spigotmc.org/wiki/spigot-installation/), [Bukkit](https://www.wikihow.com/Create-a-Bukkit-Minecraft-Server) and [Paper](https://paper.readthedocs.io/en/latest/server/getting-started.html)
```
Comment (9382) [inline README.md:14]: @ZeNyfh Comment on this please

**Review by 9382 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ZeNyfh) [reply README.md:14]: will change this to include a video instead. Will create a batch/shellscript to complete the actions for the user (placing all files into their folders, etc) making the installation alot simpler by making the server not require any proper setting up apart from agreeing to eula and starting the server with a bat/sh file (port forwarding for this is not required for something like this)

**Review by ZeNyfh — CHANGES_REQUESTED** [review]
remove links from installation

**Comment by ojokenobi** [comment]
I have acted up the reviews, check recent commits to this branch

**Review by 9382 — APPROVED** [review]
Looks fine for me

**Review by ZeNyfh — APPROVED** [review]

**Pull request merged**